lang/m2/libm2/Terminal.mod

```diff
@@ -116,5 +116,6 @@ BEGIN
 (* dtrg: changed so that instead of opening /dev/tty, fd 0 is always used. *)
```

Comment:
This comment could use an update.

Reply:
I added a commit with the comment
`(* kernigh: sent output to fd 1 *)`